### DISCUSSION TITLE
TextField aria-errormessage

### DISCUSSION THREAD

**Discussion by ghost** [adobe/react-spectrum#3526] [Q&A] [answered]
Is there any reason that useTextField doesn't make use of the "aria-errormessage" attribute when supplied with an errorMessage? Because it always maps to "aria-describedby" i'm currently thinking of doing something like this, but just wondering if there is a reason against it.

```
 const {
      inputProps: {
        'aria-describedby': ariaDescribedBy,
      },
      labelProps
      descriptionProps
      errorMessageProps
    } = useTextField(
      {
        description,
        errorMessage,
        ...
      },
      ref
    );

    const textFieldAriaProps = {
      'aria-errormessage': isErrored ? ariaDescribedBy : undefined,
      'aria-describedby':
        isDescribed && !isErrored ? ariaDescribedBy : undefined,
    };
```

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-errormessage

**Comment by devongovett** [comment, marked as answer]
aria-errormessage is a fairly new property and does not yet have wide support. https://a11ysupport.io/tech/aria/aria-errormessage_attribute
